client/src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from "react";
+import Header from "./components/Header/Header";
 import GlobalStyles from "./GlobalStyles";
 
 function App() {
   return (
     <div>
       <GlobalStyles />
+      <Header />
       <h1>Hello</h1>
```

Comment:
I Think the h1 is something, which will be later found in the main, right?

Reply:
Yes, it was just a test-h1 in the React App :)